epub34/authoring/index.html

```diff
@@ -221,12 +221,23 @@ <h4>Relationship to HTML</h4>
 						the various changes to HTML and the technologies it references to ensure they keep their
 						processes up to date.</p>
 
+					<p>The [[html]] standard also combines what used to be two separate technologies: [[?html401]] and
+						[[?xhtml11]]. These two standards differed not only in their syntaxes, but also in what elements
+						and attributes authors could use, as well as how the syntaxes were identified (their
+							<code>DOCTYPE</code> declarations).</p>
+
+					<p>The [[html]] standard now defines a single content model with rules for expressing this tag set
+						using either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered
+						HTML) or the <a data-cite="html#the-xhtml-syntax">XML syntax</a> (what was known as XHTML). Up
+						until [[epub-33]], EPUB 3 only supported the XML syntax via what were called "XHTML content
+						documents." It now allows the authoring of both syntaxes under the common name of [=HTML content
+						documents=].</p>
+
```

Comment:
I think we should be careful: the paragraph reads as if this was a done deal, and a cursory look at the draft might raise eyebrows among those who intend to complete our survey.

I think we should add an editorial issue reference to #2715, and possibly to the questionnaire.

Also, if we agree on the removal of the paragraph above, adding an editorial note (here or somewhere else, I do not mind) making it explicit that what we call "XHTML Content Document" is _not_ referring to XHTML 1.1.

Reply:
> making it explicit that what we call "XHTML Content Document" is _not_ referring to XHTML 1.1

I've added this to the issue box in the last commit, but explaining this difference becomes even less relevant long term if we drop "xhtml content document". I'm not sure where it would meaningfully fit.

The only thought I've had would be to add a "differences from epub 2" section somewhere to briefly overview the major changes. But that might make the most sense as an appendix in the overview. Since we already have the revision history stuff there, maybe I'll see if I can combine the two.

Reply:
I think it is o.k. as it is for now.

Whether we will keep the "XHTML Content Document" terminology, or whether we would drop the "X": I am not sure yet. Editorially, it is probably the good move. Keeping the "X", with appropriate warnings, may be socially better, avoiding scaring people who might think that the requirement is to use HTML _and not_ XML...

Let us cross the bridge when we get there.